kasa/cli.py

```diff
@@ -129,9 +129,17 @@ def _device_to_serializable(val: SmartDevice):
 @click.option(
     "--json", default=False, is_flag=True, help="Output raw device response as JSON."
 )
+@click.option(
+    "--show_unsupported",
```

Comment:
As this is only used for discovery, let's move the option to the discovery command.

Reply:
I've renamed the option but if I move it to the discovery command I don't see how it can be used with the kasa command which goes directly to the cli.

Reply:
See `alias` or `raw_command` for an example. Or do you mean that then it would not work when not executing `kasa discover` but just `kasa`? In that case I think it's just fine not to be able to pass it in, maybe the output printout should say "Use kasa discover --show-unsupported to see them" to make it clear that the command needs to be named explicitly.

Reply:
Ok that's done now